experimental/fRIP/README.md

```diff
@@ -1 +1,133 @@
+# fRIP Protocol
 
+**_This protocol calls for 10 million cells and yields enough crosslinked lysate for 2 IPs (i.e., 5 million cells of lysate per IP)._**
+
+## Day 1 - Crosslinking and harvesting
+1. For adherent cells, start by scraping your cells into media. Then, transfer your cells to a 50-mL conical tube on ice.
+
+- *NOTE: If you're working with adherent cells and you want to count them directly, you'll likely want to harvest them by trypsinization instead of scraping to break up cell clumps.*
+
+2. Spin down cells at 500 x g for 5 minutes at 4°C. Discard supernatant
+3. Resuspend pellet in 10 mL ice-cold PBS and transfer to 15-mL conical tube on ice
+4. Spin down cells at 500 x g for 5 minutes at 4°C. Discard supernatant
+5. Resuspend pellet in 2 mL ice-cold PBS
+6. Add 5.6 uL of 37% formaldehyde (final concentration of 0.1%)
+7. Put tube on tilt board or rotator at room temperature for 10 minutes
+8. Add 100 uL 2.5 M glycine (final concentration 125 mM)
+9. Put tube on tilt board at room temperature for 5 minutes
+10. Spin down cells at 500 x g for 5 minutes at 4°C. Discard supernatant
+11. Resuspend pellet in 2 mL PBS + 1x Roche cOmplete Protease Inhibitor Cocktail (Roche 11873580001)
+
+- *NOTE: Add PI cocktail to PBS fresh shortly before use.*
+- *NOTE: For adding the PI cocktail here and all other steps where it is used in this protocol, I work from a 25x stock (e.g., dissolve one PI tablet in 2 mL water), which I store at -20°C.*
+
+12. Spin down cells at 500 x g for 5 minutes at 4°C. Discard supernatant
+13. Resuspend pellet in 1 mL ice-cold PBS+Roche Protease Inhibitor Cocktail and transfer to eppendorf tube on ice
+14. Spin down cells at 500 x g for 5 minutes at 4°C. Discard supernatant
+15. Flash-freeze pellet in liquid nitrogen, and store at -80°C.
+
+## Day 2 - Lysing, shearing, filtering, and pre-clearing lysate
+1. Thaw crosslinked cell pellet, and resuspend in 1 mL cold **RIPA+ Buffer** on ice
+
+- 50 mM Tris pH 8
+- 150 mM KCl
+- 0.1% SDS
+- 1% Triton-X
+- 5 mM EDTA
+- 0.5% sodium deoxycholate
+- **_0.5 mM DTT_**
+- **_1x Roche cOmplete Protease Inhibitor Cocktail_**
+- **_100 U/mL RNaseOUT (Invitrogen 10777019)_**
+- (add **_these_** components fresh shortly before use)
+
+2. Rotate samples at 4°C for 10 minutes
+3. Sonicate samples at 4°C in Bioruptor on the high setting using a 30 seconds on/30 seconds off cycle for 15 minutes.
```

Comment:
This seems like over shearing compared to the original protocol (is this Yicheng's version)?

"
4)	Lyse for 90s on the Branson using program #20 (0.7s on, 1.3s off, 10% for 30s)
a.	Let the probe sit for 30s between rounds to prevent it from heating
See supplemental figure 1 and 2 where this shearing conditions and buffers were highly optimized.